packages/core/src/InstanceSettingsConfig.ts

```diff
@@ -9,4 +10,19 @@ export class InstanceSettingsConfig {
 	 */
 	@Env('N8N_ENFORCE_SETTINGS_FILE_PERMISSIONS')
 	enforceSettingsFilePermissions: boolean = false;
+
+	/**
+	 * The home folder path of the user.
+	 * If none can be found it falls back to the current working directory
+	 */
+	public readonly userHome: string;
+
+	public readonly n8nFolder: string;
+
```

Comment:
I don't think we've ever discussed this - I personally prefer not to specify `public` for public methods, enough to specify `private` ones, for less noise and for more consistency.

Reply:
no idea where that `public` came from 😅